backend/src/main/java/org/technikum/backend/model/Room.java

```diff
@@ -25,6 +27,16 @@ public class Room {
     @Column(name = "guest_capacity")
     private int guestCapacity;
 
-    @Column(name = "size_sqm")
```

Comment:
Previous decleration of sizeSqm is needed as getter setter are incompatible otherwise.